doc/users/deploy_app.rst

```diff
@@ -41,9 +41,9 @@ The ``start_test_network.sh`` script can also be used to automatically recover a
 .. code-block:: bash
 
     $ cd CCF/build
-    $ cp ./workspace/test_network_0/0.ledger .
+    $ cp -r ./workspace/test_network_0/0.ledger .
     $ cp ./workspace/test_network_0/network_enc_pubk.pem .
-    $ ../start_test_network.sh -p liblogging.enclave.so.signed --recover --ledger 0.ledger --network-enc-pubk network_enc_pubk.pem --common-dir ./workspace/test_network_common/
+    $ ../start_test_network.sh -p liblogging.enclave.so.signed --recover --ledger-dir 0.ledger --network-enc-pubk network_enc_pubk.pem --common-dir ./workspace/test_network_common/
```

Comment:
```suggestion
    $ ../start_test_network.sh -p liblogging.enclave.so.signed --recover --ledger-dir ./ledger --network-enc-pubk network_enc_pubk.pem --common-dir ./workspace/test_network_common/
```

Reply:
This is actually the ledger of the first node that gets copied locally (line `44`) before being used for recovery. I think it is better to keep the original ledger folder name here to demonstrate that we are recovering the ledger of a specific node.

Reply:
Ah, I hadn't realised 0.ledger was a directory, since the name matched the old files.